### ISSUE TITLE
Classification-only release?

### ISSUE THREAD

**Issue by tsalo** [nipreps/fmripost-aroma#39]
I am still stuck on warping to requested output spaces, but we do have ICA-AROMA classification working, so I wonder if we should make an initial release with just that part. @effigies WDYT?

**Comment by effigies** [comment]
Sounds good to me. Sorry, I haven't kept up with the state of everything here.

If you can do the following:

```
fmriprep $BIDS $FMRIPREP participant --output-space MNI152NLin6Asym ...
fmripost-aroma $BIDS $AROMA participant -d $FMRIPREP
```

And get the regressors and non-aggressive denoised BOLD in MNI152NLin6Asym, then I think we've recovered everything of import from the old fMRIPrep. Anything else is gravy.

**Comment by tsalo** [comment]
Oof... I don't have the denoised BOLD in MNI152NLin6Asym working yet. It shouldn't be too hard to get that working though.

**Comment by effigies** [comment]
What's the output currently look like?

**Comment by tsalo** [comment]
It's just the MELODIC mixing matrix, MELODIC component weight maps, ICA-AROMA component metrics and classifications, and noise components in a confounds file.

**Comment by effigies** [comment]
Screenshot or file listing?

**Comment by tsalo** [comment]
Here's the tree:
```
├── dataset_description.json
├── logs
│   ├── CITATION.bib
│   ├── CITATION.html
│   ├── CITATION.md
│   └── CITATION.tex
├── sub-01
│   ├── ses-01
│   │   ├── figures
│   │   │   ├── sub-01_ses-01_task-rest_space-MNI152NLin6Asym_desc-about_bold.html
│   │   │   ├── sub-01_ses-01_task-rest_space-MNI152NLin6Asym_desc-aroma_bold.svg
│   │   │   ├── sub-01_ses-01_task-rest_space-MNI152NLin6Asym_desc-metrics_bold.svg
│   │   │   └── sub-01_ses-01_task-rest_space-MNI152NLin6Asym_desc-summary_bold.html
│   │   └── func
│   │       ├── sub-01_ses-01_task-rest_desc-aroma_metrics.json
│   │       ├── sub-01_ses-01_task-rest_desc-aroma_metrics.tsv
│   │       ├── sub-01_ses-01_task-rest_desc-melodic_timeseries.tsv
│   │       ├── sub-01_ses-01_task-rest_space-MNI152NLin6Asym_desc-melodic_components.nii.gz
│   │       └── sub-01_ses-01_task-rest_space-MNI152NLin6Asym_desc-melodic_mixing.tsv
│   └── ses-02
│       ├── figures
│       │   ├── sub-01_ses-02_task-rest_space-MNI152NLin6Asym_desc-aroma_bold.svg
│       │   └── sub-01_ses-02_task-rest_space-MNI152NLin6Asym_desc-metrics_bold.svg
│       └── func
│           ├── sub-01_ses-02_task-rest_desc-aroma_metrics.json
│           ├── sub-01_ses-02_task-rest_desc-aroma_metrics.tsv
│           ├── sub-01_ses-02_task-rest_desc-melodic_timeseries.tsv
│           ├── sub-01_ses-02_task-rest_space-MNI152NLin6Asym_desc-melodic_components.nii.gz
│           └── sub-01_ses-02_task-rest_space-MNI152NLin6Asym_desc-melodic_mixing.tsv
├── sub-01_anat.html
├── sub-01_ses-01_func.html
└── sub-01_ses-02_func.html
```

**Comment by effigies** [comment]
That looks reasonable as-is. I think if you wanted to postpone saving the denoised time series to another release, that would be fine.

**Comment by tsalo** [comment]
It was pretty easy to write out the denoised MNI152NLin6Asym derivatives, so I just merged a PR to do that.

**Comment by tsalo** [comment]
I think we just need a job to build and deploy the Docker image.

**Comment by tsalo** [comment]
I just released 0.0.1.
